### ISSUE TITLE
Added Data Analyst Roadmap to GitHub Repository

### ISSUE THREAD

**Issue by worldofvarun** [WeMakeDevs/roadmaps#532]
Hi there! I noticed that the GitHub repository currently does not have a roadmap for aspiring data analysts. As someone who has gone through the process of becoming a data analyst, I believe that it would be incredibly helpful for newcomers to have a clear and concise roadmap to follow.

Note: this roadmap is created by me based on my learning path

**Comment by bsuryab** [comment]
Excel: Start with Excel as it provides a solid foundation for data analysis. Learn functions, formulas, data manipulation, and visualization techniques.

SQL: SQL is crucial for data retrieval and manipulation from databases. Learn how to write SQL queries to extract, clean, and transform data.

Data Cleaning and Wrangling: Understand data cleaning techniques, data preprocessing, and dealing with missing values, outliers, and inconsistencies in datasets. Learn how to transform raw data into a usable format.

Once you cover the above, If you want to dig deeper then you can focus on Python/R 

In a nutshell, understand the business and follow the below flow 

Data Source --> Transform the data --> Create a data model --> Visualizations to give the insights.
